.github/workflows/release.yml

```diff
@@ -65,7 +65,19 @@ jobs:
       with:
         gradle-executable: "./test_runner/gradlew"
         arguments: "-p test_runner clean build shadowJar"
-        
+
+    - name: Gradle Integration Tests Android
+      uses: eskatos/gradle-command-action@v1
+      with:
+        gradle-executable: "./integration_tests/gradlew"
+        arguments: "-p integration_tests test --tests IntegrationTests.shouldMatchAndroidSuccessExitCodeAndPattern -Dflank-path=../test_runner/build/libs/flank.jar -Dyml-path=./src/test/resources/flank_android.yml"
+    
+    - name: Gradle Integration Tests iOS
+      uses: eskatos/gradle-command-action@v1
+      with:
+        gradle-executable: "./integration_tests/gradlew"
+        arguments: "-p integration_tests test --tests IntegrationTests.shouldMatchIosSuccessExitCodeAndPattern -Dflank-path=../test_runner/build/libs/flank.jar -Dyml-path=./src/test/resources/flank_ios.yml"
```

Comment:
if we want to add more integration tests. is this line going to get longer and longer?

Reply:
You can also define all paths in the test and run it by class or run all. But maybe in our purposes, these parameters are redundant? WDYT?

Reply:
I think what I am trying to discuss is, is our goal here to run 1big test that does all the integration test? or lots of smaller tests so we could simply add test after test after test.

Reply:
This has been answered 👍